src/components/dashboard/uploadButton.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import IconButton from '@mui/material/IconButton'
+import AddCircleOutlinedIcon from '@mui/icons-material/AddCircleOutlined'
+import styles from '../../styles/dashboardStyles.module.css'
```

Comment:
No need to import react like this
```suggestion
import IconButton from '@mui/material/IconButton'
import AddCircleOutlinedIcon from '@mui/icons-material/AddCircleOutlined'
import styles from '../../styles/dashboardStyles.module.css'
```